### DISCUSSION TITLE
sometimes wine games not close with qtile kill shortcut

### DISCUSSION THREAD

**Discussion by ghost** [qtile/qtile#3887] [Ideas]
![2022-10-10-064127_1280x1012_scrot](https://user-images.githubusercontent.com/80429360/194796417-291bf030-5d3d-42f0-91d1-e6cee4f85dd5.png)

**Comment by m-col** [comment]
Qtile only requests that clients close their windows, we don't force anything. Bad clients that don't respond to those requests probably need to be killed some other way, such as with `pkill` or `xkill`.

**Reply by ghost** [reply]
ok
